rfcs/0038-rosetta-construction-api.md

```diff
@@ -298,7 +298,7 @@ type payment = {
 type transaction = stakeDelegation | payment
 ```
 
-Note that our client-sdk only has support for signing payments and delegations but this version of the construction API only supports those transactions types as well.
+Note that our client-sdk only has support for signing payments and delegation but this version of the construction API only supports those transactions types as well.
```

Comment:
think this should remain "delegations"

Reply:
On line 68, there's still a misspelling of "delegation", though.

Reply:
okay